src/test/java/uk/gov/hmcts/reform/pip/subscription/management/errorhandling/exceptions/PendingMigrationScriptException.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.pip.subscription.management.errorhandling.exceptions;
```

Comment:
Hhhmm this class looks just like the one in the main folder. Why do we need it here? Is that supposed to be a test?

Reply:
Good spot! It was indeed meant to be a test. I've updated this now :)